### ISSUE TITLE
How to customize keys?

### ISSUE THREAD

**Issue by Bradw-17** [fabriziofiorucci/Streamdeck#2]
Is there anyway to set up a single key to act as a hotkey? Eg. Button 1= CTRL+C, Button 2= CTRL+V, etc. (of course with a release at the end so it isn't just holding those keys down).

Been trying to figure out how to rework the code for this but im pretty much a noob.

**Comment by Bradw-17** [comment]
Also I believe there is slightly different format for Windows key codes

**Comment by Bradw-17** [comment]
So Windows wants the format "HID_KEYBOARD_XXXXX"
It appears VOLUME_UP and VOLUME_DOWN don't work however.
Just need to figure out how to make hotkeys...

**Comment by fabriziofiorucci** [comment]
Hi, the Keypad and HID project libraries I used can be found at:

https://github.com/Chris--A/Keypad
https://github.com/NicoHood/HID

I tested and am running the sample sketch using the streamdeck with a Linux PC, so it is very likely that under windows keycodes have to be changed. In the HID library github repo there are a few examples on how to use key modifiers such as the CTRL key, definitions such as KEY_RIGHT_CTRL and KEY_LEFT_CTRL can be used to send specific keystrokes via the HID interface.

I hope this helps!
